src/AutoRest.CSharp/Common/Input/Configuration.cs

```diff
@@ -264,6 +270,11 @@ internal static (string AbsoluteProjectFolder, string RelativeProjectFolder) Par
         public static bool SkipSerializationFormatXml { get; private set; }
         public static bool DisablePaginationTopRenaming { get; private set; }
 
+        public static bool AssertModelConstructorParametersForReadOnlyProperties { get; private set; }
```

Comment:
What happens if we stop asserting here for the libraries that currently do?  This doesn't feel like a breaking change.  I wonder if we should choose a single best practice here and use it everywhere.

Reply:
Consolidated. Flag is not needed anymore.